cluster-api/src/test/java/io/aiven/klaw/clusterapi/services/ApacheKafkaAclServiceTest.java

```diff
@@ -0,0 +1,254 @@
+package io.aiven.klaw.clusterapi.services;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.aiven.klaw.clusterapi.UtilMethods;
+import io.aiven.klaw.clusterapi.models.ClusterAclRequest;
+import io.aiven.klaw.clusterapi.models.enums.*;
+import io.aiven.klaw.clusterapi.utils.ClusterApiUtils;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateAclsResult;
+import org.apache.kafka.clients.admin.DeleteAclsResult;
+import org.apache.kafka.clients.admin.DescribeAclsResult;
+import org.apache.kafka.common.KafkaFuture;
+import org.apache.kafka.common.acl.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class ApacheKafkaAclServiceTest {
+  @Mock private ClusterApiUtils clusterApiUtils;
+
+  @Mock private AdminClient adminClient;
+
+  @Mock private KafkaFuture<Collection<AclBinding>> kafkaFutureCollection;
+
+  @Mock private DescribeAclsResult describeAclsResult;
+
+  @Mock private DeleteAclsResult deleteAclsResult;
+
+  @Mock private AccessControlEntry accessControlEntry;
+
+  @Mock private CreateAclsResult createAclsResult;
+
+  @Mock private KafkaFuture<Void> kFutureVoid;
+
+  private UtilMethods utilMethods;
+
+  private ApacheKafkaAclService apacheKafkaAclService;
+
+  @BeforeEach
+  public void setUp() {
+    apacheKafkaAclService = new ApacheKafkaAclService(clusterApiUtils);
+    utilMethods = new UtilMethods();
+  }
+
+  @Test
+  public void loadAcls1() throws Exception {
```

Comment:
```suggestion
  public void loadAclsScenario() throws Exception {
```

may be rename the methods to be more descriptive, like if failure, why

Reply:
These tests were earlier in other test classes, I just moved them to the relevant classes. I will check the tests and change the name accordingly for these.